fs/grub/disk_io.c

```diff
@@ -128,30 +128,31 @@ int filemax;
 int
 rawread (int drive, int sector, int byte_offset, int byte_len, char *buf)
 {
+	/* FIXME: Simplify this function, as BootIdeReadSector() supports offset and buffering */
```

Comment:
If this refactors this function anyway, why not fix this now?

Reply:
Same as above, see the notes from PR description.

Reply:
Same as https://github.com/XboxDev/cromwell/pull/27#discussion_r362119878

Reply:
Resolved in https://github.com/XboxDev/cromwell/pull/27#discussion_r362119878